RISC-V_cva6/uart/uart.c

```diff
@@ -52,6 +52,9 @@
 #define IIRF_RX_LINE_STATUS             0x06u
 #define IIRF_DATA_TIMEOUT               0x0Cu
 
+ 
+//#define PLIC_DEMO 1
+// Source/portable/GCC/RISC-V/portASM.S:290:freertos_risc_v_application_interrupt_handler (weak function : User application can add there own override)
```

Comment:
comment spelling issue:
User application can add their own override
The spell checker must have inserted the wrong version of 'their'